config/index.js

```diff
@@ -99,9 +102,15 @@ function readRcConfig() {
  */
 function pickConfigFieldsFromEnvVars(configFields) {
 	return configFields.reduce((out, key) => {
+
 		if (ENV_VARS[key] in process.env) {
 			const val = process.env[ENV_VARS[key]];
 
+			if(isWholePositiveNumber(val)){
```

Comment:
Are you sure about this logic? I might be paranoid, but if we really want to be precise we can't rely on format of the value to determine data type of the value. Currently it would work. What if we had other props that would fit `/\d+/`, but would still have to be a string? :/ Perhaps it's better to have a list of numeric values and do conversion only for those.